lib/paratii.ipfs.uploader.js

```diff
@@ -295,6 +295,9 @@ class Uploader extends EventEmitter {
           }
 
           switch (commandStr) {
+            case 'transcoding:error':
+              ev.emit('transcoding:err', argsObj.err)
```

Comment:
probbly want to call this error. Merging and fixing later if this was breaking something :-)